project/src/main/java/com/google/sps/data/TextSearchLibrary.java

```diff
@@ -46,15 +47,16 @@
     return result;
   }
 
-  enum Match {
-    EXACT, CLOSE, PARTIAL, NONE;
+  public enum Match {
```

Comment:
I think it would be really helpful to have an explanation of what the different values mean, especially how to distinguish between `CLOSE`, `PARTIAL`, and `WEAK`.

Alternatively you could have a comment explaining the steps in the algorithm at a high level, if it's easier to explain in that context.